pkg/operator/ceph/csi/spec.go

```diff
@@ -283,6 +285,7 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		if err != nil {
 			return fmt.Errorf("failed to start cephfs plugin daemonset: %+v\n%+v", err, cephfsPlugin)
 		}
+		k8sutil.AddRookVersionLabelToDaemonSet(cephfsPlugin)
 	}
 
 	if cephfsProvisionerSTS != nil {
```

Comment:
you need to add this to provisioner statefulset also

Reply:
> you need to add this to provisioner statefulset also

Resolved.